lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -1669,6 +1669,14 @@ protected function generateFieldMappingPropertyDocBlock(array $fieldMapping, Cla
                 $options[] = '"unsigned"=true';
             }
 
+            if (isset($fieldMapping['options']['fixed']) && $fieldMapping['options']['fixed']) {
+                $options[] = '"fixed"=true';
+            }
+
+            if (isset($fieldMapping['options']['default'])) {
+                $options[] = '"default"="'. $fieldMapping['options']['default'].'"';
```

Comment:
Missing spaces before and after the concatenation operator (it should be `'"default"="' . $fieldMapping['options']['default'] . '"';`). Alternatively you can also use `sprintf()` if you prefer...